TimePad/Controllers/Home/HomeViewController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  HomeViewController.swift
+//  TimePad
+//
+//  Created by Emre Alpago on 10.09.2022.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
+    
+    @IBOutlet private var taskLabel: UILabel?
+    @IBOutlet private var moreButton: UIButton?
+    @IBOutlet private var cardView: UIView?
+    @IBOutlet private var cardHourLabel: UILabel?
+    @IBOutlet private var cardDetailsButton: UIButton?
+    @IBOutlet private var todayLabel: UILabel?
+    @IBOutlet private var seeAllButton: UIButton?
+    @IBOutlet private var workTableView: UITableView?
+    
+    let cellSpacingHeight: CGFloat = 16.0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        setupColor()
+        setupCardView()
+        setupTableView()
+    }
+    
+    func setupColor() {
+        view.backgroundColor = ColorConstant.shared.mainBackgroundColor
+        cardView?.backgroundColor = ColorConstant.shared.cardBackgroundColor
+        taskLabel?.textColor = ColorConstant.shared.titleTextColor
+        cardHourLabel?.textColor = ColorConstant.shared.cardHourColor
+        todayLabel?.textColor = ColorConstant.shared.cardBackgroundColor
+        seeAllButton?.tintColor = ColorConstant.shared.cardBackgroundColor
+    }
+    
+    func setupCardView() {
+        cardView?.layer.masksToBounds = true
+        cardView?.layer.cornerRadius = cardView!.frame.width/30.0
+        cardView?.layer.borderWidth = 0.3
+    }
+    
+    func setupTableView() {
+        workTableView?.dataSource = self
+        workTableView?.delegate = self
+        workTableView?.register(UINib(nibName: TableViewConstant.shared.cellNibName, bundle: nil), forCellReuseIdentifier: TableViewConstant.shared.cellReusIdentifier)
+    }
+}
+
+extension HomeViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 10
```

Comment:
bunlara magic number diyoruz. Neden 10? Bir anlam ifade etmiyor. O zaman bunu bir constant'a çıkarıp adını anlaşılabilir bir şey koyabilirsin. Swift 5'ten sonra tek satır kodlara return koymamıza gerek yok

```
enum Constant {
    static let myDearVariableCount = 10
}
```

```suggestion
    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        Constant.myDearVariableCount
```